src/SparseInverseSubset.jl

```diff
@@ -42,15 +42,19 @@ end
 """
 Construct sparse Julia versions of L, D, U, and P from a CHOLMOD Cholesky factorization.
 """
-function get_ldup(F::SuiteSparse.CHOLMOD.Factor)
-    L = sparse(F.L)
-    P = F.p
-    d = Vector(diag(L))
-    L = tril(L * Diagonal(1 ./ d), -1)
-    U = sparse(L')
-    d = d.^2
-    D = Diagonal(d)
-    return (L=L, D=D, U=U, P=P)
+function get_ldup end
+
+if Base.USE_GPL_LIBS
```

Comment:
Should this conditional go outside the function definition?

Reply:
up to you really. I did it this way so the function will still be documented even when `Base.USE_GPL_LIBS` is `false`